keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
> In my suggestion, it's not that the user policies aren't stored in the DB, they're just made part of the blob of data mapped to the user's index and not stored separately.

I'm kind of confused here. Why do we need to store the user policies to both db and the data blob mapped to the user's index? 
Another question, do we want to preserve the user's privacy (in this case, it is the user policies) from the lookup operation? Could we just return all the data blob to the user and the user policies would be exposed; or do we have to do some kind of filtering/trimming the data blob before returning to the client? If we follow the latter way, then it also cause some overhead, much like querying the db, right?